src/hubdata/app.py

```diff
@@ -73,12 +83,15 @@ def print_dataset_info(hub_path):
         schema_lines.append(f'- [green]{field.name}[/green]: [bright_magenta]{field.type}[/bright_magenta]')
 
     # create the dataset group lines
-    num_files = sum([len(child_ds.files) for child_ds in hub_ds.children])
-    file_types = ', '.join([child_ds.format.default_extname for child_ds in hub_ds.children])
+    filesystem_datasets = hub_ds.children if isinstance(hub_ds, pa.dataset.UnionDataset) else [hub_ds]
+    num_files = sum([len(child_ds.files) for child_ds in filesystem_datasets])
+    found_file_types = ', '.join([child_ds.format.default_extname for child_ds in filesystem_datasets])
+    admin_file_types = ', '.join(hub_connection.admin['file_format'])
     num_rows = hub_ds.count_rows()
     dataset_lines = ['\n[b]dataset[/b]:',
                      f'- [green]files[/green]: [bright_magenta]{num_files:,}[/bright_magenta]',
-                     f'- [green]types[/green]: [bright_magenta]{file_types}[/bright_magenta]',
+                     f'- [green]types[/green]: [bright_magenta]{found_file_types} (found) | {admin_file_types} (admin)'
```

Comment:
I understand what "found" and "admin" mean, but not sure I understand why you want to output that information. What's the idea behind showing that information? Because it might be confusing to the user without context I think.

Reply:
Good question, and I wondered as I added the feature whether it would be confusing. The reason I added it is because I wanted to see what the app was actually finding on the cloud vs. what is "allowed" in admin.json. Maybe I should just take that info out and leave it as it was (i.e., just showing found). ?